main.go

```diff
@@ -398,6 +400,10 @@ func main() {
 		if err := rg.UpdateTasks(reqIds); err != nil {
 			log.Fatal(err)
 		}
+	default:
+		fmt.Printf(`Invalid command "%s"\n`, command)
```

Comment:
Sorry, I do not see which space you're referring to. 

I did realize that the string literal just prints out `\n` so I'll remove that.